### ISSUE TITLE
Tpot auto ML does not work when classifying more than 50 classes

### ISSUE THREAD

**Issue by amrrashed** [EpistasisLab/tpot#1146]
Tpot auto ML does not work when classifying more than 50 classes.
it appears the error could not broadcast...
to solve this issue you need to edit tpot.py file at line 58 and change 50 to the required number of classes

**Comment by weixuanfu** [comment]
This is related to a bug found in #1142. I fixed it via PR #1143  and it was merged to development branch. And It will be included in next release of TPOT.

For testing the development branch, you may install TPOT with patch into your environment via:
```
pip install --upgrade --no-deps --force-reinstall git+https://github.com/EpistasisLab/tpot.git@development
```

**Comment by weixuanfu** [comment]
This issue should be fixed in recent release (v0.11.7) of TPOT. Please update TPOT and feel free to reopen this issue if it still exists.
